eng/common/scripts/Prepare-Release.ps1

```diff
@@ -49,6 +49,7 @@ param(
 Set-StrictMode -Version 3
 
 . ${PSScriptRoot}\common.ps1
+. ${PSScriptRoot}\apiview-helper.ps1
```

Comment:
I've been trying to start to separate helper functions from the others and put them under the helpers folder can you please put this there as well.

Reply:
done